components/speedreader/rust/lib/src/readability/tests/lib.rs

```diff
@@ -64,7 +64,7 @@ mod test {
 //test!(engadget);
 //test!(folha);
 //test!(gmw);
-test!(guardian_1);
```

Comment:
this test has been failing since before my change. these tests aren't very good. they just have output and expected htmls, which will break at any change in the algorithm. i moved unit tests into the extractor.rs file, and tested specific features of the algorithm.